frontend/install-protoc.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sudo apt install protobuf-compiler
```

Comment:
https://github.com/google/android-cuttlefish/blob/main/frontend/debian/control#L5

Move it into `Build-Depends` in control file.

https://github.com/google/android-cuttlefish/blob/main/docker/Dockerfile#L20

And docker file as well.

Reply:
https://github.com/google/android-cuttlefish/blob/8927f04c23dd7264e90df78125bb110b7d7d88ae/.github/workflows/presubmit.yaml
as well